governance/roles.md

```diff
@@ -202,41 +209,104 @@ that role for some time informally, then proposed as an on-going role by Pull
 Request which must be approved by a majority of Chairs.  Process clarifications
 are welcome and PRs can be approved by any single Chair.
 
-### Security review facilitator
+### Security Assessment Facilitator
 
 [Security Reviews](/assessments) are part of the ongoing work of the group
-and led by a Security Assessment Facilitator, who will:
-
-* coordinate security review leads for upcoming security assessments.
-* identify and recommend security reviewers.
-* contribute to process improvements.
-* review and merge PRs in the /assessments directory (ensuring co-chair review
-  of significant process changes).
-* triage issues related to security assessments.
-
-### Meeting facilitator
+and led by a security assessment facilitator (referred to in the rest of this
+sub-section as a facilitator). The facilitator is effectively responsible for
+getting new security assessments underway, and providing support on an as-needed
+basis to the security assessment team if anything is impeding the overall review
+process.
+
```

Comment:
I think the part of this document like @ultrasaurus recommended should be kept brief and pointing to other documents for details. These should be high-level bullet points of what the high level roles and responsibilities are - and the how can be pointed to in the document. A lot of the content that you have here is good, but may be better to move them to the assessment guide page.

An example for illustration of a high level bullet point is:
- Ensure that security reviews follow the assessment process
- Help bootstrap security assessments and reviews
- Determine assessment schedule of reviews, considering TOC requirements
- ...

If needed, you could add a section in the guide on this topic as well.